docs/source/index.rst

```diff
@@ -52,27 +52,33 @@ threshold proxy re-encryption scheme to provide cryptographic access control.
 How does NuCypher work?
 -----------------------
 
+.. image:: ./.static/img/nucypher_overview.svg
+    :target: ./.static/img/nucypher_overview.svg
+
 1. Alice, the data owner, grants access to her encrypted data to
 anyone she wants by creating a policy and uploading it to
 the NuCypher network.
 
-2. Using her policy's public key, any entity can encrypt data on Alice's behalf.
+2. A group of Ursulas, which are nodes of the NuCypher network,
+receive information about the policy, called a PolicyArrangement that include
+a re-encryption key share. The Ursulas stand ready to re-encrypt data in exchange for payment
+in fees and token rewards. Thanks to the use of proxy re-encryption,
+Ursulas and the storage layer never have access to Alice's plaintext data.
+
+3. Using a configurable asymmetric encryption key, any entity (Enrico) can encrypt data on Alice's behalf.
```

Comment:
"a configurable asymmetric encryption key" ... that's a weird sentence. I'd say something like:

"Each policy created by Alice has associated a policy public key, which can be used by any entity to encrypt data on Alice's behalf."

Reply:
I know @jMyles has troubles with the phrase "policy public key".

Reply:
I know, but we still don't have a solution for that. I don't want to halt this just because of that. We can always change it later.

Reply:
Yeah him and I have chatted about that for the blog posts - I was trying to avoid `policy public key`.

How about:
"Each policy created by Alice has an associated [encryption] key, which can be used by any entity to encrypt data on Alice's behalf."

Reply:
That's perfect. I'd simply not use the brackets, "associated encryption key" is fine.

Reply:
✔️